src/BenchmarksServer/Startup.cs

```diff
@@ -834,17 +816,29 @@ private static string GetDotNetExecutable(string dotnetHome)
             // Force Kestrel server urls
             process.StartInfo.Environment.Add("ASPNETCORE_URLS", serverUrl);
 
-
-            if (database.HasValue)
+            if (job.Database != Database.None)
             {
-                process.StartInfo.Environment.Add("Database", database.Value.ToString());
-            }
+                string connectionString = null;
 
-            if (!string.IsNullOrEmpty(sqlConnectionString))
-            {
-                process.StartInfo.Environment.Add("ConnectionString", sqlConnectionString);
-            }
+                switch (job.Database)
+                {
+                    case Database.PostgreSql:
+                        connectionString = Environment.GetEnvironmentVariable("POSTGRES_CONN");
```

Comment:
I would pass these in as command-line parameters rather than environment variables for consistency and discoverability.

Reply:
We can't pass the values in the server command-line as each job can use a different one. So I assume you don't mean that.

We shouldn't pass it to the benchmarked application as command-line as we might have encoding issues while generating the command-line (double quotes) and we can load the database configuration more easily with `new ConfigurationBuilder().AddEnvironmentVariables()`
But if you really prefer this way then I'll do the changes and parse the command line, which we don't do at all right now in benchmarks.

Reply:
I think the BenchmarksServer should have 3 command-line parameters like `postgresqlConnectionString`, `mysqlConnectionString`, and `sqlserverConnectionString`.  Currently, BenchmarksServer takes all input as command-line parameters rather than environment variables.  Command-line parameters are more discoverable since they are listed in `--help` documentation, easier to find in the source code, etc.

No change to how it should be passed to the benchmarked application.